tools/gen-intrinsics-specs.py

```diff
@@ -21,14 +21,14 @@
 import csv
 import doctest
 
-
-def quote_literal(val):
+def quote_literal(val, table_format):
     if len(val) > 0:
-        return f"``{val}``"
+        return f"``{val}``" if table_format == "grid" else f"```{val}```"
     return val
 
 
 rst_header_levels = ['=', '~', '-', '_', '@']
+md_header_levels = ['#', '##', '###', '####', '#####']
```

Comment:
I think you could merge these two into a dictionary, say `header_levels`, to avoid the use of 

```
rst_header_levels if table_format == "grid" else md_header_levels
```

This way, you can simplify the selection removing the `...if...else...` ternary operator and just use `header_levels[table_format]`